### PR TITLE
feat: add property `z-index` to `CSnackbar` and `CSnackbarManager`

### PR DESCRIPTION
**Pull request by ItsPierre**

### CODE REVIEW

File: src/components/CSnackbar.vue [hunk 1]
```diff
@@ -70,11 +71,19 @@ const props = withDefaults(
      * Timeout in milliseconds of this snackbar after which it will be hidden.
      */
     timeout?: number | "infinite" | null;
+
+    /**
+     * The z-index used for the component.
+     *
+     * Defaults to `2500` to be above dialogs (with a z-index of `2400`). (Vuetify default: `2000`)
+     */
+    zIndex?: string | number;
```
Comment (johannes-huther) [inline src/components/CSnackbar.vue:80]: Why can this be a string?

File: src/components/CSnackbarManager.vue [hunk 2]
```diff
@@ -67,6 +68,11 @@ const props = defineProps<{
    * Whether newly added snackbars should successively be shown as soon as the currently active snackbar is closed or not.
    */
   successively?: boolean;
+
+  /**
+   * The z-index used for the component.
+   */
+  zIndex?: string | number;
```
Comment (johannes-huther) [inline src/components/CSnackbarManager.vue:75]: Same here

**Review by johannes-huther — CHANGES_REQUESTED** [review]

File: src/components/CSnackbar.vue (hunk 1 above)
Comment (ItsPierre) [reply src/components/CSnackbar.vue:80]: I don't know, I used the same definition as Vuetify: https://vuetifyjs.com/en/api/v-snackbar/#props-z-index

File: src/components/CSnackbarManager.vue (hunk 2 above)
Comment (ItsPierre) [reply src/components/CSnackbarManager.vue:75]: see above

**Review by johannes-huther — APPROVED** [review]

**Pull request merged**